app/src/main/java/com/hover/stax/channels/ChannelDropdownViewModel.java

```diff
@@ -162,6 +162,7 @@ protected void setActiveChannelIfNull(List<Channel> channels) {
 	}
 
 	private void setActiveChannel(Channel channel) {
+		FirebaseMessaging.getInstance().subscribeToTopic(getApplication().getString(R.string.firebase_topic_institution, String.valueOf(channel.institutionId)));
```

Comment:
Still not right. There is a function called logChoice that already subscribes to firebase channels

Reply:
Actually we don't want to subscribe to Institutions anyway. Institutions don't have a country, so it doesn't really help us. By subscribing users to a channel we can always connect to their institution if we want to